owncloudApp/src/main/res/values/strings.xml

```diff
@@ -518,22 +518,22 @@
     <string name="actionbar_select_inverse">Select inverse</string>
 
     <string name="actionbar_move">Move</string>
-    <string name="file_list_empty_moving">Nothing in here. You can add a folder!</string>
```

Comment:
This was intended to sound more user-friendly, so I would let it stay as it is 😄

Reply:
A friendly command? I don't think so.

Reply:
Well, just as we do with `You do not have permission to add content here!`, also reviewed by you in this PR